source/dashboard.rst

```diff
@@ -62,3 +62,48 @@ Clone Github repository
 -----------------------
 
 See :doc:`github`.
+
+Organizing your repositories with categories
+--------------------------------------------
+
+Over time, the list of repositories you deal with can grow long.  As it does, it can become difficult to
+find a repository you are looking for in the "Recent repositories" list when you have opened many other
+repositories since you opened the one you are looking for.  To help find repositories quickly, you can
+group your repositories into categories that will display in alphabetical order on the Dashboard.
+
+Create new category
+^^^^^^^^^^^^^^^^^^^
+
+While the Dashboard is being displayed (after application start-up), you create a new repository category
+by doing the following:
+
+- right click the repository you want to group into a category,
+- select "Categories",
+- select "Add new..." from the resulting fly-out menu,
+- type the name of the new category,
+- click the [OK] button.
+
+The originally-right-clicked repository will now additionally appear in your new category (repository group),
+where you can quickly find it again later no matter where it appears in the "Recent repositories" list.
+
+Rename a category
+^^^^^^^^^^^^^^^^^
+If you later decide you want a category to have a different name:
```

Comment:
insert newline after header

Reply:
Okay!  Done for both sub-headings in subsequent commit.